### PR TITLE
Fixes #289 - added Slack channel information to getting started and contact page

### LINKED ISSUE #289: Add links to slack channel and registration
We already have a reference to the mailing list in the getting started/useful links, we can add links for slack here:

https://incore.ncsa.illinois.edu/doc/incore/getting_started.html#useful-links

### PR DESCRIPTION
**Pull request by navarroc**
I added the slack channel link and slack registration link in two places, 1) Getting Started page under "Useful Links" and 2) Contact Information page

Here is the Getting started page screenshot:
getting-started-slack

Here is the Contact Information page screenshot:
contact-info-slack

### CODE REVIEW

**Review by ywkim312 — APPROVED** [review]

File: manual_jb/content/contact.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Contact Information
 
-* Contact [NCSA](http://www.ncsa.illinois.edu/) team via <incore-dev@lists.illinois.edu> regarding use of IN-CORE. The team will try to respond within 24 hours during business days.
+* You can contact the [NCSA](http://www.ncsa.illinois.edu/) team via <incore-dev@lists.illinois.edu> regarding
+  the use of IN-CORE. The team will try to respond within 24 hours during business days.
+* You can also reach out through our Slack channel. To get started, go to
+  [https://in-core.slack.com/](https://in-core.slack.com/). Or, you can grab an invitation here: [https://join.
+  slack.com/t/in-core/shared_invite/zt-25zffgnae-h0v8uGjpSli1YYp0Ypr68Q](https://join.slack.com/t/in-core/shared_invite/zt-25zffgnae-h0v8uGjpSli1YYp0Ypr68Q)
```
Comment (longshuicy) [inline manual_jb/content/contact.md:7]: I see an empyt space between "." and "slack" somehow 
image

**Review by longshuicy — APPROVED** [review]

File: manual_jb/content/contact.md (hunk 1 above)
Comment (navarroc) [reply manual_jb/content/contact.md:7]: The link works correctly so I can't tell if the space is actually there or is the browser is correcting the link. I'll move the link to a single long line so there's definitely no space.

Comment (ywkim312) [reply manual_jb/content/contact.md:7]: the link worked

Comment (navarroc) [reply manual_jb/content/contact.md]: Fixed

**Pull request merged**